plugin_template.rb

```diff
@@ -396,15 +440,18 @@ def source_paths
       cd path/to/application
       bundle
 
-  Workarea Platform Documentation
+  Features
   --------------------------------------------------------------------------------
 
-  See [http://developer.workarea.com](http://developer.workarea.com) for Workarea platform documentation.
+  ### TODO
 
-  Copyright & Licensing
+  Workarea Platform Documentation
   --------------------------------------------------------------------------------
 
-  Copyright Workarea #{Time.now.year}. All rights reserved.
+  See [https://developer.workarea.com](https://developer.workarea.com) for Workarea platform documentation.
+
+  License
+  --------------------------------------------------------------------------------
 
-  For licensing, contact sales@workarea.com.
+  Workarea #{name.titleize} is released under the [Business Software License](LICENSE)
```

Comment:
this won't actually link anywhere fwiw, would need to be the full URL like `(https://github.com/workarea-commerce/$PLUGIN/blob/master/LICENSE)`. maybe this could just link back to the platform's `LICENSE` file to keep things easy?

Reply:
We are doing this in every plugin and it's [workin' juuuuust fine](https://help.github.com/en/articles/about-readmes#relative-links-and-image-paths-in-readme-files).